cmd/beaker/dataset.go

```diff
@@ -294,20 +294,20 @@ func newDatasetRenameCommand() *cobra.Command {
 		Short: "Rename a dataset",
 		Args:  cobra.ExactArgs(2),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			dataset := beaker.Dataset(args[0])
-			if err := dataset.SetName(ctx, args[1]); err != nil {
-				return err
-			}
+			oldName := args[0]
+			newName := args[1]
 
-			info, err := dataset.Get(ctx)
+			dataset, err := beaker.Dataset(oldName).Patch(
+				ctx, api.DatasetPatch{Name: &newName},
+			)
```

Comment:
Nit-pick: This formatting is a little unusual for this code base, and its parent.

```suggestion
			dataset, err := beaker.Dataset(oldName).Patch(ctx, api.DatasetPatch{
				Name: &newName,
			})
```

Reply:
Same applies to other places with similar calls.

Reply:
Sensible. I'll adjust.